A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml

```diff
@@ -41,20 +41,20 @@
         <Sound WwiseEvent="LEFT_WING_SCRAPE" ContinuousStopDelay="0.25" FadeOutType="2" FadeOutTime="0.25" WwiseData="true" NodeName="AILERON_RIGHT001"/>
```

Comment:
Should not be here even if valid changes

Reply:
I'm not sure what you mean. `APU_FLAP_OPEN` was renamed to `A32NX_APU_FLAP_OPEN_PERCENTAGE`.